doc/source/getting_started/intro_tutorials/04_plotting.rst

```diff
@@ -131,8 +131,8 @@ standard Python to get an overview of the available plot methods:
     ]
 
 .. note::
-    In many development environments as well as ipython and
-    jupyter notebook, use the TAB button to get an overview of the available
+    In many development environments as well as IPython and
+    Jupyter notebook, use the TAB button to get an overview of the available
```

Comment:
looks like this should be capitalised too 
```suggestion
    Jupyter Notebook, use the TAB button to get an overview of the available
```
and `numpy` should be `NumPy` https://numpy.org/ . Rest looks good!